app/views/DefaultBrowserView/index.tsx

```diff
@@ -25,7 +25,7 @@ const DEFAULT_BROWSERS: IBrowsersValues[] = [
 		value: 'inApp'
 	},
 	{
-		title: isIOS ? 'Safari' : 'Browser',
+		title: 'System_default',
```

Comment:
I don't know if it's possible, but should it have an option to open Safari then?